app/static/js/update_demand_graph.js

```diff
@@ -0,0 +1,141 @@
+
+// variable to store the timeseries as the are retrieved from the database
+var timeseriesData = {}
+
+$(document).on('change', '#id_timeseries', function(event) {
```

Comment:
this could be bind to the form widget as in https://github.com/rl-institut/cp_nigeria_app/blob/dc8ebbbf47c7c1ffe64f3a0858724d656028390a/app/projects/forms.py#L945 for example